bin/diagnostic.js

```diff
@@ -1 +1,15 @@
 'use strict';
+const ask = require('../lib/ask.js');
+
+let answer = 'not empty';
+
+while (answer !== '') {
+  answer = ask("Guess my secret number between 1 and 10? ");
+  if (answer === '8') {
+   console.log("You guessed it!");
+   break;
+  }
+}
```

Comment:
First your checking to see if answer is === to ```'8'``` which is a string, it should just be ```8```. Also you forgot to ask if the number is between 1 and 10. You can check by writing either ```( answer < 1 || answer > 10)``` to check if the answer falls outside that range OR ```(answer >= 1 && answer <= 10)``` to see if answer falls inside that range.